### DISCUSSION TITLE
uv binay is missing from full container image

### DISCUSSION THREAD

**Discussion by lbarros-asml** [renovatebot/renovate#32055] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

I'm running renovate with the -full Docker image and binarySource=global and using pep621 manager with uv.lock files, but it seems the uv is missing from the full image:
```
Command failed: uv lock --upgrade-package annotated-types
/bin/sh: 1: uv: not found
```

I can't see uv here:
[renovatebot/base-image@main/Dockerfile#L24-L123](https://github.com/renovatebot/base-image/blob/main/Dockerfile#L24-L123)

Can it be added pls?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Converted to issue in base-image repo: https://github.com/renovatebot/base-image/issues/1140

PR welcome there if you're in a hurry

**Reply by lbarros-asml** [reply]
@rarkins , could you check pls:
https://github.com/renovatebot/base-image/pull/1211
